ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -86,7 +86,12 @@ private GrpcServerProtocolClient getClient() throws IOException {
   }
 
   private synchronized void resetClient(AppendEntriesRequest request, boolean onError) {
-    rpcService.getProxies().resetProxy(getFollowerId());
+    try {
+      rpcService.getProxies().getProxy(getFollowerId()).resetConnectBackoff();
+    } catch (IOException ie) {
+      LOG.warn(this + ": Failed to reset channel by " + ie);
```

Comment:
Since resetConnectBackoff() won't throw IOException, let's change the error message to
`
`      LOG.warn(this + ": Failed to getClient for " + getFollowerId(), ie);

Reply:
Done